CocoaTypograf/Operations/CancellationToken.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CancellationToken.swift
+//  CocoaTypograf
+//
+//  Created by Vadim Zhilinkov on 11/10/2018.
+//  Copyright © 2018 dreadct. All rights reserved.
+//
+
+import Foundation
+
+/// Represents a token for cancelling asynchronous operations.
+public struct CancellationToken {
+
+    // MARK: - Properties
+
+    /// An associated cancellation closure.
+    private let cancellation: () -> Void
+
+    /// Indicates whether the token was already cancelled.
+    private(set) var isCancelled: Bool = false {
+        didSet {
+            if !oldValue && isCancelled {
+                cancellation()
+            }
+        }
+    }
+
+    // MARK: - Initializers
+
+    /// Initializes a new operation token with a given cancellation closure.
```

Comment:
A mistype after renaming `OperationToken` to `CancellationToken` — "operation token" => "cancellation token".

Reply:
Fixed